CONTRIBUTING.md

```diff
@@ -3,6 +3,14 @@
 We'd love to accept your patches and contributions to this project. There are
 just a few small guidelines you need to follow.
 
+## How to build
+Fulfill [gtk-rs dependencies](http://gtk-rs.org/docs/requirements.html) for cairo crate
+
+```
+cargo build --target=x86_64-pc-windows-msvc
+```
+Only windows and macos platforms are supported at the moment
```

Comment:
readme.md starting with it's windows-only. Is that outdated or to point that macos support is weaker?

Reply:
We currently support macOS and windows approximately equally.